libsodium-sys/src/crypto_generichash.rs

```diff
@@ -70,3 +94,55 @@ fn test_crypto_generichash_primitive() {
         assert_eq!(s, crypto_generichash_PRIMITIVE.as_bytes());
     }
 }
+
+#[test]
+fn test_crypto_generichash_statebytes() {
+    assert!(unsafe { crypto_generichash_statebytes() } > 0);
+}
+
+#[test]
```

Comment:
Test vectors should definitely be added at the higher level. What I was trying to address here is that the code will compile even if the libsodium function names are misspelled. By having a simple test like this that the binding can be called, we can ensure that the functions are named correctly and do exist in libsodium.